easybuild/easyconfigs/l/libXrender/libXrender-0.9.9-intel-2015a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'libXrender'
+version = '0.9.9'
+
+homepage = "http://www.freedesktop.org/wiki/Software/xlibs"
+description = """X11 client-side library"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [XORG_LIB_SOURCE]
+
+builddependencies = [
+    ('renderproto', '0.11'),
+]
+
+dependencies = [
+    ('kbproto', '1.0.7'),
+    ('libX11', '1.6.3', '-Python-2.7.9'),
+]
+
+sanity_check_paths = {
+    'files': ['lib/libXrender.a'],
+    'dirs': [],
+}
+
+moduleclass = 'vis'
```

Comment:
What we really need is support to list multiple moduleclasses, so we can fight over which ones should be listed first. ;)